tests/v2/validation/snapshot/snapshot.go

```diff
@@ -178,9 +167,9 @@ func snapshotRestoreRKE1(t *testing.T, client *rancher.Client, podTemplate corev
 
 		clusterResp.RancherKubernetesEngineConfig.Version = etcdRestore.UpgradeKubernetesVersion
 
-		if strategy {
-			clusterResp.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableControlplane = cpUnavailalbleValue
-			clusterResp.RancherKubernetesEngineConfig.UpgradeStrategy.MaxUnavailableWorker = workerUnavailalbleValue
+		if etcdRestore.ControlPlaneUnavailableValue != "" && etcdRestore.WorkerUnavailableValue != "" {
```

Comment:
can you have one and not the other?

Reply:
Could be wrong, but I believe both are needed. If we were to manually go through this workflow, a user wouldn't intentionally leave one empty and fill only one.